internal/app/machined/pkg/controllers/network/ethernet_spec.go

```diff
@@ -29,13 +29,7 @@ func (ctrl *EthernetSpecController) Name() string {
 
 // Inputs implements controller.Controller interface.
 func (ctrl *EthernetSpecController) Inputs() []controller.Input {
-	return []controller.Input{
-		{
-			Namespace: network.NamespaceName,
-			Type:      network.EthernetSpecType,
-			Kind:      controller.InputWeak,
-		},
-	}
+	return nil
```

Comment:
Why remove this? It's still consumed

Reply:
it was a miss, controller start with a magic method which starts with waiting for udevd (different inputs), and then restores inputs, so these initial inputs are noop